packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/ports/ports.factory.js

```diff
@@ -86,10 +87,11 @@ export default /* @ngInject */ (OvhApiXdsl, $translate, TucToast) => {
         );
         return self;
       })
-      .catch(() => {
+      .catch((error) => {
         TucToast.error(
           $translate.instant('xdsl_modem_ports_add_error', {
             name: self.name,
+            reason: error.data?.message || '',
```

Comment:
It might be a good idea to consider providing a generic reason when error.data.message is empty. Otherwise no message will be shown after ':'

Reply:
it's just to avoid an error if `error.data.message` is not set by the API. It should always be a message on `error.data`.